scripts/time_average_baseline_parallelized.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
Is there a reason why this has to be a separate script, and cannot be merged with the other one? I think having fewer scripts means that there are fewer things that could potentially get out of sync, but if it makes sense to have them separate, I suppose we can live with it.

Reply:
There only needs to be one script. The non-baseline parallelized version is just the special case of the baseline parallelized version where the baseline chunk is equal to the entire set of baselines. I've just added a `cornerturn` arg that activates the baseline chunking. This also led to getting rid of a bunch of other methods in `frf`. Thanks for asking me to revisit this! The same cleanup can be used in `delay_filter` and `xtalk_filter`!